docs/supported-libraries.md

```diff
@@ -56,7 +56,7 @@ These are the supported libraries and frameworks:
 | [Eclipse Metro](https://projects.eclipse.org/projects/ee4j.metro)                                                                           | 2.2+                          | N/A                                                                                                                                                                                                                                                                                                                                                                                     | Provides `http.route` [2], Controller Spans [3]                                                                                         |
 | [Eclipse Mojarra](https://projects.eclipse.org/projects/ee4j.mojarra)                                                                       | 1.2+                          | N/A                                                                                                                                                                                                                                                                                                                                                                                     | Provides `http.route` [2], Controller Spans [3]                                                                                         |
-| [Elasticsearch API Client](https://www.elastic.co/guide/en/elasticsearch/client/java-api-client/current/index.html)                         | 7.16+                         | N/A                                                                                                                                                                                                                                                                                                                                                                                     | [Elasticsearch Client Spans]                                                                                                            |
+| [Elasticsearch API Client](https://www.elastic.co/guide/en/elasticsearch/client/java-api-client/current/index.html)                         | 7.16 - 7.17.19, 8.0 - 8.9.+   | N/A                                                                                                                                                                                                                                                                                                                                                                                     | [Elasticsearch Client Spans]                                                                                                            |
```

Comment:
maybe worth noting that versions above these ranges have native on-by-default instrumentation (so users don't think they won't get telemetry from those versions)

Reply:
added a note about this